docs/contributing/testing_guide.rst

```diff
@@ -830,8 +825,8 @@ yet able to handle correctly.
 
 To exclude a line from a coverage report, end it with
 ``# coverage: ignore``. Alternatively, we may add a line to
-``exclude_lines`` in the ``[coverage:report]`` section of
-:file:`setup.cfg` that consists of a
+``exclude_lines`` in the ``[tool.coverage.report]`` section of
+:file:`pyproject.toml` that consists of a
```

Comment:
🏕️ I noticed that our discussion of coverage settings is out-of-date. Since I was in the area, I decided to leave the campsite nicer than how I found it (hence the 🏕️).